operator/pkg/controller-runtime/cell.go

```diff
@@ -0,0 +1,93 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package controllerruntime
+
+import (
+	"context"
+	"fmt"
+	"runtime/pprof"
+
+	"github.com/bombsimon/logrusr/v4"
+	"github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	ctrlRuntime "sigs.k8s.io/controller-runtime"
+	metricsserver "sigs.k8s.io/controller-runtime/pkg/metrics/server"
+
+	"github.com/cilium/cilium/pkg/hive"
+	"github.com/cilium/cilium/pkg/hive/cell"
+	"github.com/cilium/cilium/pkg/hive/job"
+	ciliumv2 "github.com/cilium/cilium/pkg/k8s/apis/cilium.io/v2"
+	"github.com/cilium/cilium/pkg/k8s/client"
+)
+
+// Cell integrates the components of the controller-runtime library into Hive.
+var Cell = cell.Module(
+	"controller-runtime",
+	"Manages the controller-runtime integration and its components",
```

Comment:
nit: could we make a bit more useful description here? Who's controller runtime? Where to use? Here on in the comment above

Reply:
I changed the comment a little bit - more or less adding a link to their github project.

I don't want to add anything about _where to use it_. Currently there's no guideline that we want to use the controller-runtime for all controllers in the Cilium Operator.